src/app/main/service/habit-comments/habit-comments.service.spec.ts

```diff
@@ -0,0 +1,193 @@
+import { TestBed } from '@angular/core/testing';
+import { HttpClientTestingModule, HttpTestingController } from '@angular/common/http/testing';
+import { HabitCommentsService } from './habit-comments.service';
+import { environment } from '@environment/environment';
+import { HabitAddedCommentDTO, HabitCommentsModel } from '@global-user/components/habit/models/habits-comments.model';
+
+describe('HabitCommentsService', () => {
+  let service: HabitCommentsService;
+  let httpTestingController: HttpTestingController;
+  const url = environment.backendLink;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      imports: [HttpClientTestingModule],
+      providers: [HabitCommentsService]
+    });
+
+    service = TestBed.inject(HabitCommentsService);
+    httpTestingController = TestBed.inject(HttpTestingController);
+  });
+
+  afterEach(() => {
+    httpTestingController.verify();
+  });
+
+  it('should be created', () => {
+    const serviceNew: HabitCommentsService = TestBed.inject(HabitCommentsService);
+    expect(serviceNew).toBeTruthy();
+  });
+
+  it('should make POST request to add comment', () => {
+    const commentBody: HabitAddedCommentDTO = {
+      author: { id: 1, name: 'User', userProfilePicturePath: null },
+      id: 2,
+      text: 'Test comment',
+      modifiedDate: new Date().toISOString()
+    };
+
+    service.addComment(1, 'Test comment', 0).subscribe((commentData) => {
+      expect(commentData).toEqual(commentBody);
+    });
+
+    const req = httpTestingController.expectOne(`${url}habits/1/comments`);
+    expect(req.request.method).toEqual('POST');
+    req.flush(commentBody);
+  });
+
+  it('should make GET request to get active comments by page', () => {
+    const commentData: HabitCommentsModel = {
```

Comment:
Please for mock data, use `export const` in the `src/app/main/component/user/components/habit/mocks` folder.